comps/texttosql/langchain/requirements.txt

```diff
@@ -0,0 +1,17 @@
+docarray[full]
+fastapi
+langchain==0.2.9
```

Comment:
Can we remove the version just use the latest one?

Reply:
Because other microservice will use the latest version, but share if they can co-work in an example's pipeline.